sdk/appconfiguration/app-configuration/src/internal/helpers.ts

```diff
@@ -152,32 +165,41 @@ export function transformKeyValue(kvp: KeyValue) : ConfigurationSetting {
  * @ignore
  * @internal
  */
-export function transformKeyValueResponseWithStatusCode<T extends KeyValue & HttpResponseField<any>>(kvp: T) : ConfigurationSetting & { eTag?: string; } & HttpResponseField<any> & HttpResponseFields {
-  return normalizeResponse(kvp, <ConfigurationSetting & HttpResponseField<any> & HttpResponseFields>{
+export function transformKeyValueResponseWithStatusCode<
+  T extends KeyValue & HttpResponseField<any>
+>(kvp: T): ConfigurationSetting & { eTag?: string } & HttpResponseField<any> & HttpResponseFields {
```

Comment:
@richardpark-msft,  Somewhere else, eTag is actually etag. Can you take a look and make it consistent?

Reply:
It's only in there so I can delete it later on in normalizeResponse (ie, just for type safety so I can refer to it).

We do standardize on lowercase etag.

Reply:
Can you log an issue to track that work?

Reply:
Ah, it's already done.

https://github.com/Azure/azure-sdk-for-js/blob/b98c983e095f733351a6092cc190e860a9ca2cf3/sdk/appconfiguration/app-configuration/src/internal/helpers.ts#L202